src/session_test.cpp

```diff
@@ -269,6 +269,142 @@ TEST_F(SessionTest, RequestResponseError) {
   ASSERT_EQ(got.error.message, "Oh noes!");
 }
 
+TEST_F(SessionTest, RequestCallbackResponse) {
+  using Callback = std::function<void(dap::SetBreakpointsResponse)>;
```

Comment:
Would rename `Callback` to `ResponseCallback` for clarity.

Reply:
Done